import_data.py

```diff
@@ -0,0 +1,9 @@
+def import_data(data, sep=None):
+    with open('phone.csv', 'a+') as file:
+        if sep == None:
+            for i in data:
+                file.write(f"{i}\n")
+            file.write(f"\n")
+        else:
+            file.write(sep.join(data))
+            file.write(f"\n")
```

Comment:
Без этих строк будет работать также
```suggestion
        else:
            file.write(sep.join(data))
```